### ISSUE TITLE
Looking for Documentation of remote port 7642

### ISSUE THREAD

**Issue by burakozhan** [g4klx/MMDVMHost#820]
### Long version;
I was trying to send some local pages, just to test some old pocsag pagers. I did manage to do what I was trying but it made me wonder where I could find documentation about the port. 

Essentially MMDVMhost allows (with configuration) to be remote controlled via UDP packages. 
I can use `sudo RemoteCommand <port> page <RIC> "<message>"` to send a message. 
The binary `RemoteCommand` does the same as casting a UDP packet to the given port. So the same can be achieved with this:
`echo "page <RIC> <message>" | nc -u -q 1 127.0.0.1 <port>`  (where `nc` is net-cat, `-u` is for udp and `-q 1` to close input to nc in 1 second.) This way I can send Alphanumeric pages to a given RIC. What I cannot do is send Numeric only and Tone only messages to even older pagers. But this issue is not about that particular problem.

### tl;dr
I would like to know, if there is some documentation somewhere on how to use the commands that are input to the remote control port (7642 UDP) of MMDVMhost. There surely are other commands besides "page". 

Do I need to learn how to read C++ to read the code to be able to understand the commands ?
